brave-lists/debounce.json

```diff
@@ -34,6 +34,24 @@
     "action": "redirect",
     "param": "url"
   },
+  {
+    "include": [
+      "*://*.demdex.net/event?*"
+    ],
+    "exclude": [
+    ],
+    "action": "redirect",
+    "param": "d_rd"
+  },
+  {
+    "include": [
+      "*://*.doubleclick.net/ddm/clk/*"
+     ],
+     "exclude": [
+     ],
+     "action": "redirect",
+     "param": "l"
```

Comment:
Are you sure that's correct? I see the `l` _before_ the question mark, so I don't think it's going to show up in the query string parameters...